README.md

```diff
@@ -1,13 +1,50 @@
 # Publishing API
 
-An application to route requests between multiple content-store
-endpoints based on whether they're live or in other states.
+This is a Go application that proxies requests to multiple content-stores. Our
+use case for this is to keep two copies of the frontend of GOV.UK running: one
+which the public sees and another which is only used by people working on
+content to review work in progress.
+
+Publishing apps will talk to the publishing-api rather than the content-store.
+Normal publishing requests are forwarded to both the live and draft
+content-stores, whereas draft documents would only be forwarded to the draft
+content-store.
+
+### Dependencies
+
+- [alphagov/url-arbiter](https://github.com/alphagov/url-arbiter) - ?
```

Comment:
It seemed sensible for the url-arbiter integration to be done at this point.  Otherwise some content will be registered multiple times with the url-arbiter (once by each content-store).  This is redundant, and can also lead to race conditions with the url-arbiter uniqueness validation on path (this would only be the case for new items...).

Once this app is live and being used for all publishing to the content-store(s), we will remove the url-arbiter registration from content-store.